hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/CryptoGetAccountRecordsHandler.java

```diff
@@ -107,4 +114,24 @@ public Response findResponse(@NonNull final QueryContext context, @NonNull final
 
         return Response.newBuilder().cryptoGetAccountRecords(response).build();
     }
+
+    @NonNull
+    @Override
+    public Fees computeFees(@NonNull final QueryContext queryContext) {
+        final var query = queryContext.query();
+        final var accountStore = queryContext.createStore(ReadableAccountStore.class);
+        final var op = query.cryptoGetAccountRecordsOrThrow();
+        final var accountId = op.accountIDOrThrow();
+        final var account = accountStore.getAccountById(accountId);
+
+        final var pbjRecords = recordCache.getRecords(accountId);
```

Comment:
You can use the Stream API here, if you want to.

Reply:
Agreed. From previous experiences, we thought stream API is slow and stopped using anywhere. Because of that did not use it 😃 . Will change